extensions/BugModal/web/bug_modal.js

```diff
@@ -875,6 +875,21 @@ $(function() {
     // auto-enlarge comment area (up to its max-height)
     autosize($('#comment'));
 
+    // calling autosize.update or autosize
+    if (BUGZILLA.user.settings.autosize_comment) {
+    autosize.update($('#comment'));
+    }
+
```

Comment:
remove the above four lines, and put them up near the existing call to autosize.update($('#comment'));